### ISSUE TITLE
Add privacy policy/notice page

### ISSUE THREAD

**Issue by uniqueg** [elixir-cloud-aai/landing-page#54]
In accordance with EU [General Data Protection Regulation (GDPR)](https://gdpr-info.eu/) laws, we need to add a privacy policy page to the site. Create a static privacy policy page following this template: https://gdpr.eu/privacy-notice/, render it on route `/policy` and link to it from a new footer section `About website` under item "Privacy".

**Comment by Keshav-0907** [comment]
Hi @uniqueg, can I start working on this issue ??

**Comment by uniqueg** [comment]
Sure, I'll assign you. Thanks a lot :)

**Comment by uniqueg** [comment]
Yes, given that there doesn't seem to have been any progress in the last two months - please go ahead, I will assign you. Thanks a lot :)

**Comment by Keshav-0907** [comment]
@uniqueg  Can you please help me with this issue 

<img width="1440" alt="Screenshot 2023-03-10 at 1 37 32 PM" src="https://user-images.githubusercontent.com/91189139/224259118-a6f3ad74-39c3-46d8-99b8-ca0492cefdf9.png">

<img width="1440" alt="Screenshot 2023-03-10 at 1 37 23 PM" src="https://user-images.githubusercontent.com/91189139/224259077-288f5eb3-5b50-4b2d-895a-29078598cde8.png">

**Comment by uniqueg** [comment]
I guess @git-anurag-hub or @tanya17-05 are more capable of helping you here.

**Comment by tanya17-05** [comment]
Hi @Keshav-0907, have you added `NOTION_TOKEN` in `.env` file? You can generate notion token by taking help of `readme` of this repository.

**Comment by Keshav-0907** [comment]
> Hi @Keshav-0907, have you added `NOTION_TOKEN` in `.env` file? You can generate notion token by taking help of `readme` of this repository.

Yes, i have already done that

**Comment by tanya17-05** [comment]
@Keshav-0907, as I can see in console message that probably your notion API token is invalid. I've attached the screenshot from your messagefor your reference.
 ![IMG_20230312_121739.jpg](https://user-images.githubusercontent.com/95133237/224529046-f628abf1-a8a8-4f95-99ed-49cd360e202b.jpg)

**Comment by git-anurag-hub** [comment]
Thanks @Keshav-0907 for the PR & @tanya17-05 for the help to debug! Please ping me in case of any need!
